bundle/deploy/lock/release.go

```diff
@@ -45,10 +45,18 @@ func (m *release) Apply(ctx context.Context, b *bundle.Bundle) error {
 	log.Infof(ctx, "Releasing deployment lock")
 	switch m.goal {
 	case GoalDeploy:
-		return b.Locker.Unlock(ctx)
+		err := b.Locker.Unlock(ctx)
+		if err != nil {
+			return err
+		}
 	case GoalDestroy:
-		return b.Locker.Unlock(ctx, locker.AllowLockFileNotExist)
+		err := b.Locker.Unlock(ctx, locker.AllowLockFileNotExist)
+		if err != nil {
+			return err
+		}
 	default:
 		return fmt.Errorf("unknown goal for lock release: %s", m.goal)
 	}
+
+	return nil
```

Comment:
These changes are nops.

Reply:
Looks like I hadn't fully cleaned up after reverting the other changes. Updated.